<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QMainWindow" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>631</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="connection_page">
     <widget class="QPushButton" name="b_connect">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>300</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover 
{
background-color: rgb(114,255,10);
}

QPushButton:pressed 
{
background-color: rgb(12,255,10);
}</string>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>194</width>
        <height>119</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Ip:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="e_address"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sb_port"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Nickname:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="e_nickname"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="Password">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="password"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="key_input">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter key 1-26</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Quit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>300</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover 
{
background-color: rgb(255,49,12);
}

QPushButton:pressed 
{
background-color: rgb(255,9,12);
}</string>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Dowload/w514h5121380476868key.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>451</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Dowload/—Pngtree—oval comics with speech bubbles_4492754 (1) (2).png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="messages">
     <widget class="QTextEdit" name="text_messages">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>491</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="send_message">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>491</width>
        <height>61</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="b_send">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>250</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover 
{
background-color: rgb(114,255,10);
}

QPushButton:pressed 
{
background-color: rgb(12,255,10);
}</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_disconnect">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>290</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover 
{
background-color: rgb(255,49,12);
}

QPushButton:pressed 
{
background-color: rgb(255,9,12);
}</string>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>111</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
